rootfs/etc/s6/mariadb/run

```diff
@@ -29,18 +29,18 @@ then
   echo "DROP DATABASE IF EXISTS test;" >> ${INIT_FILE}
   echo "FLUSH PRIVILEGES;" >> ${INIT_FILE}
 
-  if [[ -n "${MARIADB_USERNAME}" && -n "${MARIADB_DATABASE}" ]]
+  NEW_DB__NAME=${MARIADB_DATABASE:-$MARIADB_USERNAME}
```

Comment:
Please set the exported database variable to this value, than this variable can get reused for scripting within the container.